doc/geany.txt

```diff
@@ -2536,10 +2536,14 @@ Tool paths
 ``````````
 
 Terminal
-    The command to execute a script in a terminal.  Occurrences of %c
+    The command to execute a script in a terminal. Occurrences of %c
     in the command are substituted with the run script name, see
     `Terminal emulators`_.
 
+File Manager
+	The command to use to open a file manager. Occurences of %d
+	are substituted with a directory in which to open it.
+
```

Comment:
Say _what_ is substituted for %d directory of current file or home dir if no files open, its up to the command to determine what to do with that.